### PR TITLE
Update Windstream Parser for new emails

### PR DESCRIPTION
**Pull request by aliex-13**
It seems Windstream have updated their emails with an additional table to include GPS location - Lat and Long.

This has messed with the existing parser so i've had to add some additional checks to the parser.

Just re-checks for maint_id, start and end time.

No other changes included.

### CODE REVIEW

File: circuit_maintenance_parser/parsers/windstream.py [hunk 1]
```diff
@@ -46,6 +46,7 @@ def parse_html(self, soup):
             if len(row) < 2:
                 continue
             cols = row.find_all("td")
+            print(cols)
```
[the remark this answers is not in the document]
Comment (aliex-13) [reply circuit_maintenance_parser/parsers/windstream.py:49]: haha woops! I have prepped a revert for this however looking to add a test for this new format before committing/pushing.

File: circuit_maintenance_parser/parsers/windstream.py [hunk 2]
```diff
@@ -64,6 +65,20 @@ def parse_html(self, soup):
...
+
+        event_start = soup.find("td", text="Event Start Date & Time:").find_next_sibling("td").string
+        if event_start:
+            dt_time = convert_timezone(event_start)
+            data["start"] = int(dt_time.replace(tzinfo=timezone.utc).timestamp())
+
+        event_end = soup.find("td", text="Event End Date & Time:").find_next_sibling("td").string
+        if event_end:
+            dt_time = convert_timezone(event_end)
+            data["end"] = int(dt_time.replace(tzinfo=timezone.utc).timestamp())
```
[the remark this answers is not in the document]
Comment (aliex-13) [reply circuit_maintenance_parser/parsers/windstream.py:80]: Yes, these lines are obsolete now. I've fixed this up and will add this to my next push.

**Comment by aliex-13** [comment]
> Can you provide an (anonymized) example of the updated email format to use as a test case?

Hey @glennmatthews - I am struggling with adding a new updated example of this test case. The reason being that the email I have received from Windstream is encoded a charset of `<meta http-equiv="Content-Type" content="text/html; charset=us-ascii">`

It's not passing my testing locally at the moment, so I will have a play around with it a bit more and see if I can work it out.

If there is any recommended workflow for getting source email into a good format, can you please share it?

**Comment by aliex-13** [comment]
Hey @glennmatthews, test email has been added and unit tests updated.
Other threads have been resolved.

**Comment by chadell** [comment]
@aliex-13 could you fix the CI tests?

**Comment by aliex-13** [comment]
Hey @chadell - I've pushed updates to fix the pylint issues but other issues unrelated to my code are occurring. Any chance you could take a look at the pipeline and let me know. I noticed these when running my own tests internally but didn't want to mess around with other code which was already in this state.

**Comment by chadell** [comment]
> Hey @chadell - I've pushed updates to fix the pylint issues but other issues unrelated to my code are occurring. Any chance you could take a look at the pipeline and let me know. I noticed these when running my own tests internally but didn't want to mess around with other code which was already in this state.

Yes @aliex-13 , going to fix the doctest

File: circuit_maintenance_parser/output.py [hunk 3]
```diff
@@ -83,7 +83,6 @@ class CircuitImpact(BaseModel, extra="forbid"):
         pydantic_core._pydantic_core.ValidationError: 1 validation error for CircuitImpact
         impact
           Input should be 'NO-IMPACT', 'REDUCED-REDUNDANCY', 'DEGRADED' or 'OUTAGE' [type=enum, input_value='wrong impact', input_type=str]
-        ...
```
Comment (chadell) [inline circuit_maintenance_parser/output.py:86]: @aliex-13 could you readd this to fix the doctest?

**Review by chadell — APPROVED** [review]
LGTM

**Pull request merged**